lib/EnhancedSwitch.js

```diff
@@ -247,9 +247,11 @@ var EnhancedSwitch = (function (_React$Component) {
       }
     }
   }, {
-    key: 'isChecked',
-    value: function isChecked() {
-      return this.refs.checkbox.checked;
+    key: 'setIndeterminate',
+    value: function setIndeterminate() {
+      if (this.props.indeterminate) {
+        this.refs.checkbox.indetermiante = true;
```

Comment:
Happy to help :) using a self made polyfill now, so I'm def looking forward to this.